drv/ArrayFIFOQueue.drv

```diff
@@ -192,13 +210,32 @@ public class ARRAY_FIFO_QUEUE KEY_GENERIC implements PRIORITY_QUEUE KEY_GENERIC,
 		length = (end = size) + 1;
 		array = newArray;
```

Comment:
instead: array = TO_KEY_ARRAY(), right? Looks like its exactly the same code (+ changes to start, length and end)

Reply:
First toArray has to be accepted by @vigna but that kinda makes sense. Though custom implementations would be more dangerous then.